src/ReactiveUI/RxApp.cs

```diff
@@ -56,14 +58,14 @@ public static class RxApp
 #endif
 
         [ThreadStatic]
-        private static IScheduler _unitTestTaskpoolScheduler;
-        private static IScheduler _taskpoolScheduler;
+        private static IScheduler? _unitTestTaskpoolScheduler;
+        private static IScheduler? _taskpoolScheduler;
         [ThreadStatic]
-        private static IScheduler _unitTestMainThreadScheduler;
-        private static IScheduler _mainThreadScheduler;
-        private static IObserver<Exception> _defaultExceptionHandler;
+        private static IScheduler? _unitTestMainThreadScheduler;
```

Comment:
Not sure if we want these to be nullable. Again probably should be set from the static constructor some how.